extensions/parquet/table/src/main/java/io/deephaven/parquet/table/pagestore/PageCache.java

```diff
@@ -7,7 +7,9 @@
 import java.lang.ref.WeakReference;
 
 /**
- * Page cache data structure.
+ * A Cache for {@link IntrusivePage IntrusivePages}. This datastructure stores pages as soft references and maintains
+ * them as an LRU cache. Externally references can be held as WeakReferences so that as memory pressure builds they can
+ * be evicted from the cache.
```

Comment:
```suggestion
 * A cache for {@link IntrusivePage IntrusivePages}. This data structure stores pages as {@link SoftReference soft references} and maintains
 * them as an LRU cache. External references to cached pages should be held via {@link WeakReference weak references} so that as memory pressure builds the pages can
 * be evicted from the cache.
```